forge/db/migrations/20241209-01-remove-dead-instance-notifications.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Remove Notifications for deleted Instances
+ */
+
+const deletedProjects = []
+const foundProjects = []
+
+module.exports = {
+    /**
+     * @param {QueryInterface} queryInterface Sequelize.QueryInterface
+     */
+    up: async (context) => {
+        const deleteNotification = async function (id) {
+            const deleteQuery = `DELETE FROM "Notifications" WHERE "id" = ${id};`
+            await context.sequelize.query(deleteQuery)
+        }
+
+        const lookup = 'SELECT "id", "reference" from "Notifications" ' +
+            'WHERE "type" = \'instance-crashed\' OR "type" = \'instance-safe-mode\';'
+        const [notifications] = await context.sequelize.query(lookup)
+        for (const notification of notifications) {
+            const projectId = notification.reference.split(':')[1]
+            if (deletedProjects.includes(projectId)) {
+                // already found project to be gone, delete
+                deleteNotification(notification.id)
+            } else {
+                if (!foundProjects.includes(projectId)) {
+                    const projectQuery = `SELECT count(*) from "Projects" WHERE "id" = '${projectId}' LIMIT 1;`
+                    const [projectExits] = await context.sequelize.query(projectQuery)
+                    if (projectExits[0] === 0) {
```

Comment:
Running this against sqlite and postgres results in different shapes of results that don't work with the test that is then performed on them

**sqlite**
```
[ [ { 'count(*)': 1 } ], Statement {} ]
```

**postgres**
```
[ [ { count: '1' } ], Result { ... } ] ]
```


In neither case will the current code correctly match the actual result. Note the different field names and the values are different types (int/string).

```suggestion
                    const projectQuery = `SELECT count(*) as c from "Projects" WHERE "id" = '${projectId}' LIMIT 1;`
                    const [[projectCount]] = await context.sequelize.query(projectQuery)
                    const count = parseInt(projectCount.c)
                    if (count === 0) {
```